client/src/components/LoadingScreen.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useTheme } from '../contexts/ThemeContextSet'
+
+export default function LoadingScreen() {
+  const {check, theme, toggleTheme } = useTheme()
+  let bgColor = 'rgb(253,253,253)'
+  let fgColor = 'rgba(0,104,249, 0.6)'
+  if(theme === 'dark'){
+    bgColor = 'rgb(29,29,29)'
```

Comment:
Some of these colors also exist in LineChart -- could they be constants in a shared file?